app/Template/task/show.php

```diff
@@ -13,3 +13,6 @@
 <?= $this->render('task/timesheet', array('task' => $task)) ?>
 <?= $this->render('file/show', array('task' => $task, 'files' => $files, 'images' => $images)) ?>
 <?= $this->render('task/comments', array('task' => $task, 'comments' => $comments, 'project' => $project)) ?>
+
+<script src="../../../assets/js/vendor/jquery-1.11.1.min.js"></script>
+<script src="../../../assets/js/keyboardShortcuts.js"></script>
```

Comment:
Do not include js files here, everything should be in the app.js, see http://kanboard.net/documentation/assets